### ISSUE TITLE
Expose more metadata fields on Item Details page

### ISSUE THREAD

**Issue by ggeisler** [sul-dlss/dlme#1369]
User research indicated that at least a couple of advanced users were confused by the search results they received for a given query. One reason (there are probably other, more complicated reasons related to Solr, etc.) might be that it seems that sometimes a record is returned where the query term is not displayed anywhere in the metadata on the item details page.

Assuming this is the case (I'm not 100% certain, but it appears that way), one thing we could do to reduce confusion is to display a fuller list of metadata fields on the item details page, to ensure that any query term matched is also visible on that page.

This could add to the length of the page, which isn't great. But we could use an approach we did in the Vatican project, where we hid additional metadata field in a collapsed section that could be opened with a "More details" button:

<img width="591" alt="Screen Shot 2021-12-13 at 5 24 52 PM" src="https://user-images.githubusercontent.com/101482/145910378-4837c52b-c673-4ba5-a1b7-b946fc13b14a.png">

When that button was selected, we exposed the full list of metadata fields (the Vatican had an unusually long list of fields so we broke them up into named sections), and changed the button label so the user understood they could go back to the shorter list:

<img width="591" alt="Screen Shot 2021-12-13 at 5 25 23 PM" src="https://user-images.githubusercontent.com/101482/145910462-1cef11a7-d2ea-47ed-9b33-6a728d3a8363.png">

**Comment by mwerla** [comment]
I agree that we should be displaying all the metadata fields (as long as we don't display duplicated data - for example, "dirty" source values and values that we processed). Regarding the "More details" button, I am wondering if it's not enough to order the metadata fields in a way that whatever would be under "More details" is simply displayed in the end. Then if the user scrolls down, they get to all the metadata. The button would make sense if there is something more on the page below (too lengthy) metadata record, but there is nothing more (besides the footer) in the DLME UI case.
